actions/image_partition_action.go

```diff
@@ -12,6 +12,7 @@ Yaml syntax:
    imagename: image_name
    imagesize: size
    partitiontype: gpt
+   uuid: string
```

Comment:
After some thought, `partitionuuid` is probably best

Reply:
Thanks for your suggestion. It looks consistent with `partitiontype` variable.